src/constants/nav.tsx

```diff
@@ -0,0 +1,52 @@
+//- Routes Imports
+import { ROUTES } from './routes';
+import { MarketIcon, DAOIcon, StakingIcon, ProfileIcon } from 'components';
+
+export const TITLES = {
+	MARKET: 'MARKET',
+	ZDAO: 'ZDAO',
+	STAKING: 'STAKING',
+	PROFILE: 'PROFILE',
+};
+
+export const getNavLinks = () => {
+	const LINKS = [
+		{
+			label: 'NFTs',
+			route: ROUTES.MARKET,
+			icon: (stroke: string) => (
+				<MarketIcon title={TITLES.MARKET} stroke={stroke} />
+			),
+		},
+		{
+			label: 'DAOs',
+			route: ROUTES.ZDAO,
+			icon: (stroke: string) => <DAOIcon title={TITLES.ZDAO} stroke={stroke} />,
+		},
+		{
+			label: 'Staking',
+			route: ROUTES.STAKING,
+			icon: (stroke: string) => (
+				<StakingIcon title={TITLES.STAKING} stroke={stroke} />
+			),
+		},
+	];
+
+	return LINKS;
+};
```

Comment:
Not sure why it should be function instead of constant in constants file.

```suggestion
export const NAV_LINKS = [
		{
			label: 'NFTs',
			route: ROUTES.MARKET,
			icon: (stroke: string) => (
				<MarketIcon title={TITLES.MARKET} stroke={stroke} />
			),
		},
		{
			label: 'DAOs',
			route: ROUTES.ZDAO,
			icon: (stroke: string) => <DAOIcon title={TITLES.ZDAO} stroke={stroke} />,
		},
		{
			label: 'Staking',
			route: ROUTES.STAKING,
			icon: (stroke: string) => (
				<StakingIcon title={TITLES.STAKING} stroke={stroke} />
			),
		},
	];
```